Package@swift-5.swift

```diff
@@ -4,6 +4,9 @@ import PackageDescription
 
 let package = Package(
     name: "CryptoSwift",
+    platforms: [
```

Comment:
it needs 'swift-tools-version:5.1', isn't it?

Reply:
Nope, platforms is supported in 5.0.